presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/ReadRowsHelper.java

```diff
@@ -14,12 +14,12 @@
 package io.prestosql.plugin.bigquery;
```

Comment:
Change the commit message to "Improve readRows performance"

Also, explain in the commit message improves performance. I see a refactoring, but I can't figure out what's the difference between the two versions that will improve performance.

Reply:
@martint This change is a port from the spark connector - instead of reading the entire stream as Avro and them converting it to Presto pages, the read from the server is done in a lazy fashion using the new Iterator. We have seen performance improvement especially when reading part of the data (for example when using `LIMIT`)

Reply:
Got it. Please add that explanation to the commit message

Reply:
Just did...